salt/orchestrate/edx/backup.sls

```diff
@@ -87,8 +112,17 @@ mount_backup_drive:
         fstype: ext4
         mkmnt: True
         opts: 'relatime,user'
-    - require:
-        - salt: format_backup_drive
+
+create_backup_directory:
+  salt.function:
+    - tgt: 'G@roles:backups and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - file.directory
+    - kwargs:
+        name: backups
```

Comment:
This should be `/backups/tmp` so that the `tmp` subdirectory is created. Also, without the leading `/` the backups directory would be wherever the process happened to be running from. Granted, that is most likely to be the `/` directory, but no guarantees.